<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountRegistrationWidget</class>
 <widget class="QWidget" name="AccountRegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="regRepPassLabel">
     <property name="text">
      <string>Repeat password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="regRepPassEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="surnameLabel">
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="surnameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>passwordRepeatChanged(QString)</signal>
  <signal>userNameChanged(QString)</signal>
  <signal>surnameChanged(QString)</signal>
  <slot>setPasswordRepeat(QString)</slot>
  <slot>setUserName(QString)</slot>
  <slot>setSurname(QString)</slot>
  <slot>resetPasswordRepeat()</slot>
  <slot>resetUserName()</slot>
  <slot>resetSurname()</slot>
 </slots>
</ui>
